packages/core-sdk/lib/Types/IConnector.ts

```diff
@@ -9,14 +9,21 @@ export interface IConnectorAdapters {
   multicall: IMulticallAdapter;
 }
 
+export interface OnNetworkChangeOptions {
```

Comment:
I would rename it to IConnectOptions as it is defining the connect method options.